brick/hooks/pre_gen.dart

```diff
@@ -1,5 +1,21 @@
 import 'package:mason/mason.dart';
+import 'package:recase/recase.dart';
 
 void run(HookContext context) {
   context.vars['current_year'] = DateTime.now().year;
+
+  context.vars['application_id_android'] = _appId(true, context);
+  context.vars['application_id_ios'] = _appId(false, context);
+}
+
+String _appId(bool android, HookContext context) {
```

Comment:
Having this as bool feels hacky, shouldn't this be an enum?  
Also, shouldn't we have a lint to avoid positional bool arguments?

Reply:
I don't see this as hacky, I need a variable to represent a binary value, an enum seems overkill to me.

But I agree on the positional argumenets, gonna name them